doc/generate-api-docs.sh

```diff
@@ -50,7 +50,8 @@ LIBS_SORTED=$(
 )
 
 for LIB in $LIBS_SORTED; do
-    cd $LIB
+    LIB_DIR=$(find . -maxdepth 2 -name ${LIB} -type d)
+    cd ${LIB_DIR}
```

Comment:
Please revert?

Reply:
This is required. Libs like `sgx-isa` have moved to `intel-sgx/sgx-isa`. Doing `cd $LIB` won't work in all cases.